### DISCUSSION TITLE
Hovering the star rating difficulty icon of the difficulty card should show the star rating too

### DISCUSSION THREAD

**Discussion by Gonzalo-Bruna** [ppy/osu#14168] [Ideas] [answered]
I couldn't find an issue tracking this, but currently hovering the diff icon on a difficulty card doesn't show anything:

![osu_2021-08-07_23-33-29](https://user-images.githubusercontent.com/41755816/128619859-b9234493-0f59-4b5e-a5c5-05eba8610aed.jpg)
![osu_2021-08-07_23-33-34](https://user-images.githubusercontent.com/41755816/128619861-540d5ded-e674-4fb6-85fc-cbe9a5aab7e7.jpg)

where in other places it does:
![osu_2021-08-07_23-33-37](https://user-images.githubusercontent.com/41755816/128619879-a8f8c7df-4ca0-48bf-91ab-54666bc4927f.jpg)

**Comment by gagahpangeran** [comment, marked as answer]
Star rating will be shown in the future design.

![image](https://user-images.githubusercontent.com/1329837/128006144-d3f4b9df-55f3-4a21-8ed5-2cdec045f02f.png)

https://www.figma.com/file/DXKwqZhD5yyb1igc3mKo1P/Song-Select-Draft-2?node-id=0%3A1

**Reply by Gonzalo-Bruna** [reply]
Fair enough, I guess I'll have to wait until these designs are implemented. Thanks!
